docs/api/models.md

```diff
@@ -29,6 +29,6 @@ neural quantum states.
    ARNNConv2D
    FastARNNConv1D
    FastARNNConv2D
-   DeepSet
+   DeepSetRelDistance
```

Comment:
this model seems a bit too specific, we should have only DeepSet with options I guess

Reply:
I agree, but until someone implements it...